build/build-image/Dockerfile

```diff
@@ -35,11 +35,12 @@ RUN rm -rf /var/lib/apt/lists/
 RUN apt-get -o Acquire::Check-Valid-Until=false update && apt-get install -y rsync
 
 # Download and symlink etcd.  We need this for our integration tests.
+ENV ETCD_VERSION v2.0.12
```

Comment:
It's important to keep this in sync with what we install in our default cluster builds.  How are the two kept in sync?